### ISSUE TITLE
Interface for frontend- backend communication

### ISSUE THREAD

**Issue by Noah-Kurz** [amosproj/amos2022ss07-nft-playbook#41]
## User story
1. As a poweruser
3. I want / need to integrate different information in the NFT
4. So that I can modify it

## Acceptance criteria
* Define parameters for API

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
